projects/rero/ng-core/src/lib/menu/menu-item.ts

```diff
@@ -106,6 +118,66 @@ export class MenuItem implements ItemInterface {
     return this;
   }
 
+  /**
+   * Get Prefix
+   * @return Object with name and class properties
+   */
+  getPrefix() {
+    return this._prefix;
+  }
+
+  /**
+   * Set Prefix
+   * @param name - string
+   * @param htmlClass - string
+   * @return MenuItem
+   */
+  setPrefix(name: string, htmlClass?: string) {
+    this._prefix = this._setPrefixSuffix(name, htmlClass);
+
```

Comment:
could you remove empty lines in these functions

Reply:
done